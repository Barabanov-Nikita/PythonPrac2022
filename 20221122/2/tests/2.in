Òåêñò äëÿ âòîðîãî òåñòà âòîðîãî çàäàíèÿ