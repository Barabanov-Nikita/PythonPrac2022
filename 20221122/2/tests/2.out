Текст для второго теста второго задания
